src/API/JSONData.hs

```diff
@@ -144,8 +138,8 @@ instance Show SpielResponse where
   show (SpielTypeHole m _ _)            = show m
   -- show a fallback error, such as reading a bad-file
   show (SpielError m)                   = show m
-  show x                                = show x
-
+  show (SpielTypeError e)               = show e
+  show _                                = "unused SpielResponse" -- todo: clean these out
```

Comment:
I did not want to write cases for unused responses, but I also did not want to remove them in this change set for fear of changing too much at once.

Reply:
That's a good call. This is a certain unspoken limit before the PR should just be split into another PR haha